api/src/main/java/org/openmrs/module/eptsreports/EptsReportsActivator.java

```diff
@@ -54,7 +55,13 @@ public void willStop() {
 	 * @see #started()
 	 */
 	public void started() {
-		reportsInitializer.initializeReports();
+		try {
+			reportsInitializer.initializeReports();
+		}
+		catch (IllegalArgumentException e) {
```

Comment:
I think `IllegalArgumentException` here does not necessarily mean that a metadata lookup failed.

Reply:
Yeah, it means there was no resource based on the look up identifier given